Autodesk/synthesis
# Analytics for API and Button Clicks

Based on team discussions and the upcoming build, I incorporated some bare minimum analytics which includes:
- Tracking Main UI buttons (e.g. MoveArrows, DeleteEntity, AddEnvironment, AddEntity, etc.)
- Exporter data (whether the model is from Fusion or Inventor)
- A clean Google Analytics project to track only data from our new modernized Synthesis here on out
_Note:_ Some API changes were made for the purpose of analytics.

**Testing**
1. Log into the autodeskbxd@gmail.com account via Google Analytics.
2. In the Google Analytics home dashboard, click on the new project "Synthesis App".
3. Navigate to the "Realtime" data and click "Events".
4. Run Synthesis to see live event updates with basic button clicking and the fulfilled exporter data.

**Example Tracking** (Confirmed in Unity Editor during Runtime and Builds)
![image](https://user-images.githubusercontent.com/16855850/94101083-ffd87200-fde3-11ea-9ac4-6b366440786d.png)

**Improvements**
Due to nature of our changing UI, I will note analytics updates that should take place in a new GitHub issue. Unfortunately, there is not a way to disable analytics specific to development.
